iroh-relay/Cargo.toml

```diff
@@ -116,7 +116,7 @@ iroh-test = { version = "0.29.0", path = "../iroh-test" }
 serde_json = "1"
 
 [features]
-default = ["metrics", "server"]
+default = ["metrics"]
```

Comment:
Oh wow, that's an oversight. Is there some way to measure how much this improves `iroh` dep count?

Reply:
it’s definitely not great, I am going to see to push this as patch release